src/libraries/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -25,7 +25,12 @@ internal sealed partial class RuntimeType : TypeInfo, ICloneable
         public override int MetadataToken => RuntimeTypeHandle.GetToken(this);
         public override Module Module => GetRuntimeModule();
         public override Type? ReflectedType => DeclaringType;
-        public override RuntimeTypeHandle TypeHandle => new RuntimeTypeHandle(this);
+        public override RuntimeTypeHandle TypeHandle
+        {
+            [Intrinsic] // to avoid round-trip "handle -> RuntimeType -> handle" in JIT
```

Comment:
The JIT won't look for `RuntimeType.get_TypeHandle` intrinsic. Do we really need this?